### PR TITLE
Windows fixes

### LINKED ISSUE #1611: Implement Kernel Interruption for Windows
It would be very nice to have this feature which is available in the classic notebook.

I've just submitted a calculation with a mistake in it which will take ~15mins to finish :grimacing: so figured I would use my time productively and open a feature request issue!

### PR DESCRIPTION
**Pull request by rgbkrk**
Closes #1611, #1939

Guess I'm going after all of @dhirschfeld's Windows bugs. :smile:

If you're available @jkornblum, it would be good to have you look this over.

### CODE REVIEW

**Comment by rgbkrk** [comment]
I'm going to go ahead and get this out the door so we can ship it along with the fix for help (`?`).

File: packages/desktop/src/main/menu.js [hunk 1]
```diff
@@ -472,6 +472,10 @@ const helpDraft = {
   role: "help",
   submenu: [
     {
+      label: "About nteract",
+      role: "about"
```
Comment (lgeiger) [inline packages/desktop/src/main/menu.js:476]: According to the electron docs, the `about` menu is only a macOS thing: https://electron.atom.io/docs/api/menu-item/#roles

File: packages/desktop/src/notebook/menu.js [hunk 2]
```diff
@@ -212,17 +212,7 @@ export function dispatchKillKernel(store) {
...
-  if (process.platform === "win32") {
-    notificationSystem.addNotification({
-      title: "Not supported in Windows",
-      message: "Kernel interruption is currently not supported in Windows.",
-      level: "error"
-    });
-  } else {
-    store.dispatch(interruptKernel);
-  }
+  store.dispatch(interruptKernel);
```
Comment (lgeiger) [inline packages/desktop/src/notebook/menu.js:215]: When was this fixed?

I remember https://github.com/nteract/hydrogen/issues/597 and the [issue on `spawn-rx`](https://github.com/paulcbetts/spawn-rx/issues/11
) is still open.

**Review by lgeiger — COMMENTED** [review]
I'm a bit worried that this won't work 😞

**Comment by dhirschfeld** [comment]
@lgeiger - see also my follow-up testing in #1964

**Pull request merged**